ansible/roles/gpu/tasks/nvidia-gpu-RHEL.yaml

```diff
@@ -9,12 +9,26 @@
     enablerepo: "{{ 'offline' if offline_mode_enabled else '' }}"
     disablerepo: "{{ '*' if offline_mode_enabled else '' }}"
 
-- name: install additional Nvidia runfile prerequisites
+- name: find installed Nvidia RPMs
```

Comment:
can we see if we need these or if including the missing library packages was enough.

Reply:
I think it's cleaner to have this approach anyway, but yeah sure I can test this tomorrow morning.

Reply:
Removed the commit with this task change as it is not necessary and tested it out to work.